pom.xml

```diff
@@ -631,6 +631,8 @@
                                 <Release>${project.version}</Release>
                                 <Build-Time>${maven.build.timestamp}</Build-Time>
                                 <URL>http://www.splicemachine.com</URL>
+                                <Spark-Major-Version>${spark.major.version}</Spark-Major-Version>
```

Comment:
This change affected all jars including those built in the _core_ profile for which _spark.major.version_ and _spark.version_ are not defined.